easybuild/easyconfigs/t/TRIQS/TRIQS-3.1.0-gofb-2021a.eb

```diff
@@ -62,11 +53,8 @@ sanity_check_commands = [
     "python -c 'import triqs'"
 ]
 
-modextrapaths = {
-    'CPLUS_INCLUDE_PATH': 'include',
-    'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages',
-    'CMAKE_PREFIX_PATH': ['lib/cmake/triqs', 'lib/cmake/cpp2py']
-}
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
 modextravars = {
     'TRIQS_ROOT': '%(installdir)s',
     'TRIQS_VERSION': '%(version)s'
```

Comment:
Are these environment variable known and used at runtime? The version can be found in `$EBVERSIONTRIQS`.

Reply:
They do not seem used at runtime. Removed.